SportsHub/SportsHub.Api/Controllers/ArticleController.cs

```diff
@@ -31,5 +31,12 @@ public async Task<ActionResult<ArticleResponseDTO>> GetArticleByTitleAsync(strin
             var articleResponse = _mapper.Map<ArticleResponseDTO>(article);
             return Ok(articleResponse);
         }
+
+        public async Task<ActionResult> DeleteArticleAsync(int id)
+        {
+            bool deletedSuccessful = await _articleService.DeleteArticleAsync(id);
+            if (deletedSuccessful) return Ok("Article deleted successful");
+            return BadRequest("Unable to delete article");
```

Comment:
Okey

Reply:
Do you want me to create constants for every request because we dont have any  and return them in all controllers not only for this scenario with delete , because in every controller we return a string not a constant

Reply:
do you have an updated branch with the main?

Reply:
i did not updated with today`s merge, but i do not have any conflicts currently

Reply:
That`s okay that you don`t have conflicts, but there were changes related to constants and request messages, I think almost every message is in constant files right now